.cfn-nag-ignore-lists.yml

```diff
@@ -0,0 +1,11 @@
+RulesToSuppress:
+  - id: W40
+    reason: The security group for vpc endpoint is generated by CDK
+  - id: W5
+    reason: The security group for vpc endpoint is generated by CDK
+  - id: W92
```

Comment:
why suppressing this rule globally? try to suppress your custom resource separately

Reply:
remove global cfn-nag rules